### PR TITLE
[Feature] Update html content for Nutzung (closes 1124)

### PR DESCRIPTION
**Pull request by i336616**
Updated html content and formatting

### CODE REVIEW

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/usage.html [hunk 1]
```diff
@@ -17,7 +17,7 @@
     }
     
     
-    a {
+    a[href] {
```
Comment (EisvogelSAP) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/usage.html:20]: Can you also update that in the en file and the `privacy-policy.html` file, please? :-)

**Review by eisvogel — APPROVED** [review]

File: src/xcode/ENA/ENA/Resources/Localization/en.lproj/usage.html [hunk 2]
```diff
@@ -1,448 +1,454 @@
 <!DOCTYPE html>
 <html lang="de">
```
Comment (steins-code) [inline src/xcode/ENA/ENA/Resources/Localization/en.lproj/usage.html:2]: Why is it german for english case?

**Review by steins-code — CHANGES_REQUESTED** [review]
Please include english case :)

File: src/xcode/ENA/ENA/Resources/Localization/en.lproj/usage.html (hunk 2 above)
Comment (EisvogelSAP) [reply src/xcode/ENA/ENA/Resources/Localization/en.lproj/usage.html:2]: I guess the whole file is not yet localized :-D

Comment (i336616) [reply src/xcode/ENA/ENA/Resources/Localization/en.lproj/usage.html:2]: Yep, leaving all the localization out of this commit!

**Comment by i336616** [comment]
> Please include english case :)

Ah, ok!

**Review by steins-code — APPROVED** [review]

**Pull request merged**